mypy-strict.ini

```
@@ -40,6 +40,8 @@ files =
     tools/autograd/*.py,
     tools/clang_tidy.py,
     tools/codegen/*.py,
+    tools/codegen/api/*.py,
+    tools/codegen/dest/*.py,
```

Rather than using a wildcard that only matches one directory deep, why not just match the entire directory?
```suggestion
    tools/codegen/api,
    tools/codegen/dest,
```
See #55700 which does a similar thing.

---

Actually, you might want to ignore this and instead use my other suggestion.